Source/compositorclient/RPI/Implementation.cpp

```diff
@@ -183,6 +184,93 @@ class Platform {
 
 class Platform {
 private:
+    static constexpr uint16_t VIDEO_LAYER = 10000;
```

Comment:
Why we used this big layer numbers ? Do we have to support this much ?

Reply:
Not really, the only reason to give a finite number instead of indefinite.

Reply:
Ok